cds_ils/migrator/relations/api.py

```diff
@@ -116,25 +116,30 @@ def migrate_document_siblings_relation():
                     document_class, legacy_pid_type, relation["related_recid"]
                 )
             except PIDDoesNotExistError as e:
-                pass
-
-            # try to find sibling in series
-            if related_sibling is None:
-                try:
-                    related_sibling = get_record_by_legacy_recid(
-                        series_class,
-                        legacy_pid_type,
-                        relation["related_recid"],
+                continue
+            rec_type = related_sibling["$schema"].split("/")[-1].split("-")[0]
+            # If it's series it must be a multipart, so we search for the
+            # first volume and link it.
```

Comment:
Maybe better to assert it than? What happens if it is not the case?
`assert series["MODE_OF_ISSUANCE"] == MULTI...`